examples/compiled/geo_repeat.vg.json

```diff
@@ -85,128 +84,158 @@
   ],
   "projections": [
     {
-      "name": "projection",
-      "size": {"signal": "[width, height]"},
-      "fit": {
-        "signal": "[child__repeat_row_population_geojson_0, child__repeat_row_engineers_geojson_0, child__repeat_row_hurricanes_geojson_0]"
-      },
+      "name": "__repeat_row_population_projection",
```

Comment:
Projections are not merged because layout sizes are not merged correctly here.

Reply:
Then fix the layout size to merge correctly? 

I don't think we should merge this without fixing it.

Reply:
Filed as https://github.com/vega/vega-lite/issues/6024 since it's orthogonal